packages/clients/meldemichel/src/plugins/Gfi/Feature.vue

```diff
@@ -89,9 +80,22 @@ export default Vue.extend({
     displayImage(): boolean {
       return this.currentProperties.pic
     },
+    currentProperties(): GeoJsonProperties {
+      return { ...this.windowFeatures[this.visibleWindowFeatureIndex] }
+    },
+  },
+  mounted() {
+    this.setActionButton({
+      component: ActionButtons,
+      props: {},
+    })
+  },
```

Comment:
This is not properly added, as it happens too late. It is only visible, if the feature is clicked a second time.

Check with the workaround in DISH, how that is handled there.

Reply:
I don't see a workaround (Meldemichel GFI does the same as DISH GFI?) and I'm currently confused as of why there's a need for one anyway. Maybe we can fix the need for one eventually. But for today, what's the arcana here?

Reply:
#55

Reply:
https://github.com/Dataport/polar/issues/55